src/pagination/PaginationList.js

```diff
@@ -241,7 +241,7 @@ class PaginationList extends Component {
         }
 
         return (
-          <PageButton key={ page }
+          <PageButton key={ index }
```

Comment:
ref: https://reactjs.org/tutorial/tutorial.html#keys

The tips from reactjs doc, I suggested to use like `${title}-${index}` to instead it ~~

Reply:
hmm.. would this make a difference since we're including the `index` anyways?

Reply:
I fixed it